src/main/java/sg/ncl/MainController.java

```diff
@@ -28,6 +28,7 @@
 import org.springframework.web.client.RestClientException;
 import org.springframework.validation.BindingResult;
 import org.springframework.web.client.RestTemplate;
+import org.springframework.web.servlet.mvc.condition.MediaTypeExpression;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.springframework.web.servlet.mvc.condition.MediaTypeExpression'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)